examples/gcoap/Makefile

```diff
@@ -73,6 +74,15 @@ ifeq (1,$(USE_ZEP))
   USEMODULE += socket_zep
 endif
 
+# Uncomment the following lines to specify static link lokal IPv6 address
+# this might be useful for testing, in cases where you cannot or do not want to
+# run a shell with ifconfig to get the real link lokal address.
+ IPV6_STATIC_LLADDR ?= '"fe80::cafe:cafe:cafe:1"'
+ CFLAGS += -DCONFIG_GNRC_IPV6_STATIC_LLADDR=$(IPV6_STATIC_LLADDR)
+# uncomment to not increment the last digit by the inderface number
+ CFLAGS += -DCONFIG_GNRC_IPV6_STATIC_LLADDR_IS_FIXED=1
+ CFLAGS += -DCONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF=4
```

Comment:
When I drop 0d4caf9ab874bbf97b1f629eb4ae1ee11171d24c, there will be.
Maybe I would also drop b282c62284294300b4deb323c5322fc3a3f5766c, because I just copied that from `gnrc_networking`. It is not much related to this PR.

Reply:
Ah sorry I didn't notice there was a [dropme] commit